support-frontend/assets/pages/subscriptions-redemption/thankYouContainer.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import { connect } from 'react-redux';
+import ThankYouContent from 'pages/digital-subscription-checkout/thankYouContent';
+import type { RedemptionPageState } from 'pages/subscriptions-redemption/subscriptionsRedemptionReducer';
+import { Stripe } from 'helpers/paymentMethods';
+
+function mapStateToProps(state: RedemptionPageState) {
+  return {
+    countryGroupId: state.common.internationalisation.countryGroupId,
+    paymentMethod: Stripe,
```

Comment:
if it's redemption it won't be Stripe, there won't be one, (in the scala we did an Either, but I'm not sure how it goes in flow)